### PR TITLE
530 bootstrap element pagination

### PR DESCRIPTION
**Pull request by Delnote**
530 bootstrap element pagination implementation and tests done.

### CODE REVIEW

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/PaginationAssert.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' is enabled")
+    public PaginationAssert disabled() {
+        jdiAssert(element.isDisabled() ? "disabled" : "enabled", is("enabled"));
+        return this;
+    }
+
+
```
Comment (mf-code) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/PaginationAssert.java]: two empty strings

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Pagination.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    @JDIAction("Select '{0}' in '{name}'")
+    public UIElement get(int index) {
+        return list().get(index - 1);
+    }
+
+    @JDIAction("Select '{0}' in '{name}'")
+    public void click(int index) {
+        list().select(index - 1); }
```
Comment (mf-code) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Pagination.java]: } should be in the new line (you can use ctrl-alt-L / cmd-alt-L to fix it by Intellij Idea automatically)

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/PaginationAssert.java (hunk 1 above)
Comment (Delnote) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/PaginationAssert.java]: Fixed.

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Pagination.java (hunk 2 above)
Comment (Delnote) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Pagination.java]: Fixed.

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/StatesTests.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        paginationStates.paginationItems.get(3)
+                .is()
+                .core()
+                .hasClass("active");
+    }
+
+    @Test (dataProvider = "listData")
+    public void linkTextTests(int index, String linkText) {
+        paginationStates.paginationItems.get(index)
+                .is().displayed().enabled()
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/StatesTests.java]: should be on different lines:
```
               .displayed()
               .enabled()
```

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/SizingTests.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        paginationSizeBig.paginationItems.is().size(3);
+        paginationSizeBig.is().core().hasClass("pagination pagination-lg");
+        paginationSizeSmall.paginationItems.is().size(3);
+        paginationSizeSmall.is().core().hasClass("pagination pagination-sm");
+    }
+
+    @Test (dataProvider = "listData")
+    public void linkTextTests(int index, String linkText) {
+        paginationSizeBig.paginationItems.get(index)
+                .is().displayed().enabled()
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/SizingTests.java]: should be on different lines:
```
               .displayed()
               .enabled()
```

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/OverviewTests.java [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    @Test
+    public void isValidationTests() {
+        paginationOverview.paginationItems.is().size(5);
+        paginationOverview.is().core().hasClass("pagination");
+    }
+
+    @Test (dataProvider = "listData")
+    public void linkTextTests(int index, String linkText) {
+        paginationOverview.paginationItems.get(index)
+                .is().displayed().enabled()
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/OverviewTests.java]: should be on different lines:
```
               .displayed()
               .enabled()
```

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/IconsTests.java [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    @Test
+    public void isValidationTests() {
+        paginationIcons.paginationItems.is().size(5);
+        paginationIcons.is().core().hasClass("pagination");
+    }
+
+    @Test (dataProvider = "listData")
+    public void linkTextTests(int index, String linkText) {
+        paginationIcons.paginationItems.get(index)
+                .is().displayed().enabled()
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/IconsTests.java]: should be on different lines:
```
               .displayed()
               .enabled()
```

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/AlignTests.java [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+    @Test(dataProvider = "listData")
+    public void linkTextTests(int index, String linkText) {
+        paginationAlignCenter.paginationItems.get(index)
+                .is().displayed().enabled()
+                .css("font-size", is("14px"))
+                .hasClass("page-item")
+                .text(is(containsString(linkText)));
+        paginationAlignEnd.paginationItems.get(index)
+                .is().displayed().enabled()
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/pagination/AlignTests.java]: should be on different lines:
```
               .displayed()
               .enabled()
```

**Pull request merged**